mpmp/src/model/Plot.java

```diff
@@ -36,6 +36,9 @@ public int getHouses() {
 		return 0;
 	}
 
+	public void setHouses(int houses) {
+	}
```

Comment:
You see, only `HousePlot`s can have houses. I didn't want to complicate the code in `Controller.plotUpdate` just for this special case, so I made this no-op method that is overridden in `HousePlot` and ignored in the other types. Maybe I should write it as `{}` or put a no-op comment there...